ImageFeed/SplashViewController/SplashViewController.swift

```diff
@@ -8,18 +8,30 @@
 import UIKit
 
 final class SplashViewController: UIViewController {
-    private let showAuthenticationScreenSegueIdentifier = "showAuthViewController"
+    private let splashLogo: UIImageView = {
+        let imageView = UIImageView(image: UIImage(named: "PracticumLogo"))
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+
+        return imageView
+    }()
 
     private let oauth2Service = OAuth2Service()
+    private let profileService = ProfileService.shared
     private let oauth2TokenStorage = OAuth2TokenStorage()
 
+    private var alertPresenter: AlertPresenterProtocol! = AlertPresenter()
```

Comment:
зачем неявный опционал, если можно без него обойтись: 
```suggestion

    private let oauth2Service = OAuth2Service()
    private let profileService = ProfileService.shared
    private let oauth2TokenStorage = OAuth2TokenStorage()

    private var alertPresenter: AlertPresenterProtocol = AlertPresenter()
```

Reply:
Заиспользовал это по привычке, для того, чтобы при использовании DI, если неправильно была зарезолвлена зависимость, приложение падало и было понятно в чем проблема. Поскольку зависимости разруливаются на старте прилажки, то в целом это является безопасным способом, хоть и не совсем корректным, наверное. В любом случае уберу неявный опционал)

Reply:
согласна, мы на этом этапе ничего про DI почти не говорим и не показываем как это лучше делать, к сожалению, поэтому любой вариант сейчас будет в пределах нормы)